### ISSUE TITLE
Create User Journeys

### ISSUE THREAD

**Issue by chenopis** [kubernetes/website#4548]
## TL;DR

This project will refactor the kubernetes.io `/docs/home/` page into a portal for user journeys, which are customized paths through the documentation for various buckets of users, such as cluster ops, app developers, DevOps transitioning from VMs/on-prem to Kubernetes, etc. [[project proposal](https://docs.google.com/a/google.com/document/d/1EdQ8acmuKGlzZy1agejLqmB7cLpTRBJKC0JYptJ-ylg/edit?usp=sharing)] [[project tracker](https://github.com/kubernetes/kubernetes.github.io/projects/5)]

The main objectives are to:

1. Optimize the onboarding experience across various types of users.
2. Give users a sense of where they are in the landscape of Kubernetes knowledge.
3. Explicitly tell users where to go next to improve their mastery.
4. Make users more confident during their onboarding experience -- they know where they are, they know where they're going. 

## Tasks

- Determine common and/or critical user buckets, pare down to a reasonable set.
- For each bucket, determine the critical path for concepts and skills.
- For each path, partition concepts, skills, and challenges into Foundation, Intermediate, and Advanced Topics.
- Create customized ToCs for each bucket.
- Tag docs w/ labels for search and arbitrary groupings.
- For docs matching key terminology, e.g. Pod, make the "Summary" section use the glossary snippet.
- Repurpose "Prerequisites" to identify to the user what concept and/or skills the current topic is built upon.
- Repurpose "What's Next" to direct users to learn the next skill or concept.
- Add new docs that fill in gaps by tying things together.
- Perform A/B tests of the new design, e.g. on consultants, at hackathons, etc.

**Comment by chenopis** [comment]
@Bradamant3 @abiogenesis-now We will need to get you into the Kubernetes org before we can assign things to you.

**Comment by chenopis** [comment]
@alexcontini This is the project I mentioned. You can see the project tracker here: https://github.com/kubernetes/kubernetes.github.io/projects/5

**Comment by abiogenesis-now** [comment]
Still in progress

/remove-lifecycle stale
